tests/test_slow_SW.py

```diff
@@ -0,0 +1,257 @@
+
+    params = params_SW()
+
+    params.Ra_side = 1.86e8
+
+    params.oper.enable_sfd = float(True)
+
+    params.nek.general.end_time = 600
+    params.nek.general.write_interval = 10
+
+    sim_sfd = Simul(params)
+
+    sim_sfd.make.exec("run_fg", resources={"nproc": 4})
+
+    sim = load(sim_sfd.path_run)
+
+    restart_file = sim.params.output.path_session / "cbox0.f00059"
+
+    params = params_SW()
+
+    params.Ra_side = 1.86e8
+
+    params.nek.general.start_from = "base_flow.restart"
+
+    params.nek.problemtype.equation = "incompLinNS"
+    params.oper.elem.staggered = "auto"
+
+    params.nek.general.extrapolation = "standard"
+
+    sim = Simul(params)
+
+    copyfile(restart_file, sim.params.output.path_session / "base_flow.restart")
+
+    sim.make.exec("run_fg", resources={"nproc": 4})
+
+    sim = load(sim.path_run)
+    coords, df = sim.output.history_points.load()
+
+    df_point = df[df.index_points == 12]
+
+    time = df_point["time"].to_numpy()
+    ux = df_point["ux"].to_numpy()
+    temperature = df_point["temperature"].to_numpy()
+
+    step = np.where(time > 700)[0][0]
+
+    time = time[step:]
+    ux = ux[step:]
+
+    signal = ux
+
+    c = argrelmax(signal)  # local maxima
+
+    time_loc_max = time[c]
+
+    signal_loc_max = signal[c]
+    signal_loc_max = np.ma.array(signal_loc_max)
+
+    slope, intercept, r_value, p_value, std_err = stats.linregress(
+        time_loc_max, np.log(signal_loc_max)
+    )
+
+    growth_rate_L = slope
+
+    assert 0.002 < growth_rate_L < 0.006
+
+    # if everything is fine, we can cleanup the directory of the simulations
+    rmtree(sim_sfd.path_run, ignore_errors=True)
+    rmtree(sim.path_run, ignore_errors=True)
+
+
+@pytest.mark.slow
+def test_SW_linear_base_provided():
+
+    params = params_SW()
+
+    restart_file = "./doc/examples/base_flow_side_simple.restart"
+
+    params = params_SW()
+
+    params.Ra_side = 1.86e8
+
+    params.nek.general.start_from = "base_flow.restart"
+
+    params.nek.problemtype.equation = "incompLinNS"
+    params.oper.elem.staggered = "auto"
+
+    params.nek.general.extrapolation = "standard"
+
+    sim = Simul(params)
+
+    copyfile(restart_file, sim.params.output.path_session / "base_flow.restart")
+
+    sim.make.exec("run_fg", resources={"nproc": 4})
+
+    sim = load(sim.path_run)
+    coords, df = sim.output.history_points.load()
+
+    df_point = df[df.index_points == 12]
+
+    time = df_point["time"].to_numpy()
+    ux = df_point["ux"].to_numpy()
+    temperature = df_point["temperature"].to_numpy()
+
+    step = np.where(time > 700)[0][0]
+
+    time = time[step:]
+    ux = ux[step:]
+
+    signal = ux
+
+    c = argrelmax(signal)  # local maxima
+
+    time_loc_max = time[c]
+
+    signal_loc_max = signal[c]
+    signal_loc_max = np.ma.array(signal_loc_max)
+
+    slope, intercept, r_value, p_value, std_err = stats.linregress(
+        time_loc_max, np.log(signal_loc_max)
+    )
+
+    growth_rate_L = slope
```

Comment:
What does mean `_L`? Generally, names have to be meaningful. `growth_rate` might be sufficient.